quicklogic/pp3/tests/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+add_custom_target(all_ql_tests_bit_v)
+add_custom_target(all_ql_tests_bit)
+add_custom_target(all_ql_tests_prog)
+
+add_dependencies(all_ql_tests_prog all_ql_tests_bit)
+
+add_custom_target(all_ql_tests)
```

Comment:
I think this target should be called alongside with `all_quicklogic_tests` in the script that runs the [quicklogic tests](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/ce096809b67596e77ac4ad2241a562729c943d38/.github/kokoro/ql.sh#L27) so that it gets checked as well.

Or better, I think that there should be only one target that runs tests for all quicklogic devices.

Reply:
I added dependency for target that tests the pp3 sample designs (`all_ql_tests`) so it will be executed together with `all_quicklogic_tests` in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/2196/commits/8639b1c71cd4f0340778b6e71ef4e9a868d33c7a